getting-started/clis/calicoctl/install.md

```diff
@@ -17,17 +17,336 @@ You can run `calicoctl` on any host with network access to the
 For step-by-step instructions, refer to the section that
 corresponds to your desired deployment.
 
-- [As a binary on a single host](#install-calicoctl-as-a-binary-on-a-single-host)
+<!--- Change download URL to latest release if user browsing master branch.  --->
+{%- if page.version == "master" -%}
+{% assign version = "latest/download" %}
+{% else %}
+{% assign version = "download/" | append: site.data.versions.first.components.calicoctl.version %}
+{% endif %}
 
-- [As a container on a single host](#install-calicoctl-as-a-container-on-a-single-host)
+## Install calicoctl as a binary on a single host
 
-- [As a Kubernetes pod](#install-calicoctl-as-a-kubernetes-pod)
+{% tabs %}
+<label:Linux,active:true>
+<%
+1. Log into the host, open a terminal prompt, and navigate to the location where
+you want to install the binary.
 
-- [As a kubectl plugin](#install-calicoctl-as-a-kubectl-plugin-on-a-single-host)
+   > **Tip**: Consider navigating to a location that's in your `PATH`. For example,
+   > `/usr/local/bin/`.
+   {: .alert .alert-success}
 
+1. Use the following command to download the `calicoctl` binary.
 
-{% include content/ctl-binary-install.md %}
+   ```bash
+   curl -o calicoctl -O -L  "https://github.com/projectcalico/calicoctl/releases/{{ version }}/calicoctl" 
+   ```
 
-{% include content/ctl-container-install.md %}
+1. Set the file to be executable.
 
-{% include content/ctl-kubectl-plugin-install.md cli="calicoctl" codepath="/calicoctl" %}
```

Comment:
I think it is better to write everything in this page instead of loading stuff scattered all over the place.

@caseydavenport WDYT?

Reply:
That's my preference as well. Let me ask around to see if there's a reason it needs to be done the way it is currently.